msg/odlc_type.msg

```diff
@@ -0,0 +1,2 @@
+bool standard
```

Comment:
Can be represented as only one bool, `is_emergent`, or could be `int8` like the others